assets/README.md

```diff
@@ -1,58 +1,154 @@
 
 ## License
 Please note that the fonts and the svg-icons are licensed only for the usage on websites that are **owned by the Commonwealth of Massachusetts**. Mayflower is currently using a licensed web font Texta and purchased [Smashing Icons](https://smashicons.com/) for some icons.
 
-### Texta License
+## Texta License
 Fontspring grants Licensee a perpetual, worldwide, non-exclusive and non-transferrable license to install the Texta as webfont on Websites that are owned by the Commonwealth of Massachusetts using the @font-face selector in CSS files. For other usage, you may have to acquire an additional license through Fontsprint.
 For more details about Texta's terms of use, please refer to our [web font license](https://www.fontspring.com/lic/htswufoczd).
 
-## Publish Package
+## Publishing Instructions
+>To publish the package, you will have to be a collaborator or have access to the [@massds/assets npm package](https://www.npmjs.com/package/@massds/assets).
 
-1. `npm login` — Login to the npm account.
->To publish the package, you will have to be a collaborator or have access to the [@massds/mayflower-tokens npm package](https://www.npmjs.com/package/@massds/mayflower-tokens).
+1. Once you're done making changes and ready to publish, run `npm run build`. The provided gulp file will copy over all fonts, images, and scss files to the `dist/` directory for you.
+2. `npm login` — Login to the npm account.
+3. Update `version` in package.json and run `npm publish`. Wait a few minutes and check [@massds/assets](https://www.npmjs.com/package/@massds/assets) on the npm registry.
 
-1. `npm run build` — Build the distribution folder `dist`.
 
-1. Update `version` in package.json.
-1. `npm publish` — Publish `@massds/mayflower-tokens` to npm registry.å
-1. Wait a few minutes and check [@massds/mayflower-tokens][npm] on the registry.
 
+## Setup and Usage (fonts/images)
+If you're using this package only for its fonts and images, you can skip installing bourbon and sass. Assets can be found under `dist/fonts` and `dist/images`.
 
-[npm-badge]: https://img.shields.io/npm/v/@massds/mayflower-react.png?style=flat-square
-[npm]: https://www.npmjs.com/package/@massds/mayflower-tokens
-[mayflower-github]: https://github.com/massgov/mayflower
-[mayflower-doc]: https://www.mass.gov/mayflower
-[react-storybook]: https://mayflower.digital.mass.gov/react
-[react-starter]: https://github.com/massgov/mayflower-react-starter
-[patternlab]: https://mayflower.digital.mass.gov/patternlab
-[shared-assets]: https://github.com/massgov/mayflower/tree/develop/assets
+## Setup and Usage (Sass)
+When using this package for the .scss files, you should set the sass package's include paths to include bourbon's include paths as well as this package's own. This package's include paths are:
```

Comment:
Should we setup these packages as peer dependencies:
- sass
- bourbon
- bormalizer?